MANIFESTO

```diff
@@ -1,62 +1,62 @@
-[Note: this is the Redis manifesto, for general information about
-       installing and running Redis read the README file instead.]
+[Note: this is the Valkey manifesto, for general information about
+       installing and running Valkey read the README file instead.]
```

Comment:
I almost want to delete this file instead of renaming it. This was the Redis manifesto, it feels wrong to keep it since we aren't Redis anymore and I don't know we intend to copy it. I almost kind of would like to keep it with a note at the top like this:

```suggestion
[Note: This was the manifesto of Redis. It does not represent the ideals of Valkey, but is
       kept in remembrance for the ideals that Salvatore had for the project.] 
```

Reply:
reverted